nexus/Freqlog/Freqlog.py

```diff
@@ -327,6 +326,7 @@ def __init__(self, backend_path: str, password_callback: callable, loggable: boo
         self.backend: Backend = SQLiteBackend(backend_path, password_callback, upgrade_callback)
         self.q: Queue = Queue()
         self.listener: vinput.EventListener | None = None
+        self.listener_thread = Thread(target=lambda: self._log_start())
```

Comment:
Nit: `target=self._log_start, args=(self,)` is cleaner and more efficient than a lambda

Reply:
From @xslendix:
> i cannot get the args thing to work
> for the threading.Thread
> tried both
> ```
> self.listener_thread = Thread(target=self._log_start, args=(self,))
> ```
> and
> ```
> self.listener_thread = Thread(target=self._log_start, args=(self))
> ```
> ill keep the lambda apprach since it's simple and works just fine